src/version.h

```diff
@@ -0,0 +1,3 @@
+#define passgen_VERSION_MAJOR 1
```

Comment:
Why change this file??

Reply:
Sorry about that, I meant to add #pragma once to version.h.in

Reply:
No, remove this, we don't need it here.